config/nrfconnect/chip-gn/.gn

```diff
@@ -22,5 +25,5 @@ default_args = {
   target_cpu = "arm"
   target_os = "zephyr"
 
-  import("//args.gni")
+  import("${chip_root}/config/nrfconnect/chip-gn/args.gni")
```

Comment:
nit: if I read the GN documentation correctly `import("args.gni")` should be enough.

Reply:
This does not work with `--dotfile` argument.